src/components/AccountCard.js

```diff
@@ -25,6 +25,13 @@ class AccountCard extends PureComponent<Props> {
     const { onPress, account, style, disabled } = this.props;
     const currency = getAccountCurrency(account);
     const unit = getAccountUnit(account);
+
+    const availableOnCompound =
+      account.type === "TokenAccount" && !!account.compoundBalance;
+    const balance = availableOnCompound
+      ? account.spendableBalance
```

Comment:
mmh I'm not sure about the logic here, why are we not doing `account.spendableBalance` for all? I thought it was equal to balance in non compound case already.

Reply:
Ha ok. I was not sure if spendableBalance was available all the time